chart/flux/templates/deployment.yaml

```diff
@@ -183,3 +196,4 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
     {{- end }}
+  {{- end }}
```

Comment:
This `{{ - end }}` doesn't appear to match anything

Reply:
Done.